spec/tiny_server.rb

```diff
@@ -78,12 +80,13 @@ def url
     end
 
     def block_until_started
```

Comment:
unfortunately @jkeiser this is not an option in this case (but a nice trick to stash in my memory banks). `block_until_started` is not intended to block until the thread completes but to block until the webrick server is started. So we "want" the thread to run until the tests finish.